package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "airplay-js": "guerrerocarlos/node-airplay-js",
     "application-config": "^0.2.1",
+    "async": "^2.0.0-rc.5",
```

Comment:
Let's not use `async`. It's a fatty dependency. I have better packages, `run-parallel`, `run-series`, `run-auto`, `run-waterfall`, etc. Most of those are already in our dependency tree from webtorrent, so let's stick with them.